Library/Homebrew/standalone/init.rb

```diff
@@ -24,6 +24,13 @@
   vendored_versions.include?("#{ruby_major}.#{ruby_minor}")
 end.freeze
 
+# TODO: remove `respond_to?` check when required Ruby >= 3.4
```

Comment:
Can you stick a `odeprecated` or `odisabled` in here? Makes it easier to `grep` for when doing a major/minor update.